lib/folio/mhld_builder.rb

```diff
@@ -30,7 +30,10 @@ def build
     # Remove suppressed record and electronic records
```

Comment:
# Remove suppressed records, electronic records, and records with no holdings statement